helper/src/config.json

```diff
@@ -438,6 +439,7 @@
                                 }
                             ],
                             "azurepolicy": "deny",
+                            "bastion": true,
```

Comment:
This looks like its still in the addons section?

Reply:
Also, if it goes into one of the security cards, it needs to go into all 3.   For example, If the user clicks from high security to no security, the change should be reversed out if possible

Reply:
Good spot 👍 
Moved it.